turbopack/crates/turbopack-ecmascript/src/code_gen.rs

```diff
@@ -94,9 +114,63 @@ impl CodeGenerationHoistedStmt {
 }
 
 pub trait VisitorFactory: Send + Sync {
-    fn create<'a>(&'a self) -> Box<dyn VisitMut + Send + Sync + 'a>;
+    fn create<'a>(&'a self) -> Box<dyn AstModifier + 'a>;
+}
+
+pub trait PassFactory: Send + Sync {
```

Comment:
This is temporarily required for refactoring, and it will be removed with https://github.com/vercel/next.js/pull/79555